internal/reconcilers/repository.go

```diff
@@ -152,6 +153,16 @@ func (r *Reconciler) handleArtifactsReconcilerEvent(ctx context.Context, evt *Re
 	for _, artifact := range artifacts {
 		// store information if we do not have it
 		typeLower := strings.ToLower(artifact.GetPackageType())
+		externalID, err := proto.Marshal(&pb.GitHubArtifactID{
```

Comment:
Same comment as for pull requests: ideally we should factor this out into a dedicated struct/interface... but now is not the time.